youtube_dl_gui/locale/es_ES/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -219,12 +216,12 @@ msgstr "Estado"
 
 #: youtube_dl_gui/mainframe.py:235
 msgid "Get URL"
-msgstr ""
+msgstr "Buscar dirección URL"
```

Comment:
@MrS0m30n3 You are correct, took a look at the source code. It copies the currently selected item to the clipboard. Sorry about that, missed the context, I thought it could find the video by title now.